qa/rpc-tests/test_framework/script.py

```diff
@@ -794,7 +795,7 @@ def __repr__(self):
         # need to change
```

Comment:
based on your change, this comment should also be removed.

Reply:
Very true. Done. I have meanwhile also added a very short test of this to be executed with `py.test`. There's a lot of code in the test frame work that isn't really run or tested in itself at all so I think it makes sense to test the test frame work. Especially if it becomes an extra library down the road.